os/arch/arm/src/amebasmart/amebasmart_mipi.c

```diff
@@ -304,7 +312,17 @@ static int amebasmart_mipi_transfer(FAR struct mipi_dsi_host *dsi_host, FAR cons
 	FAR struct amebasmart_mipi_dsi_host_s *priv = (FAR struct amebasmart_mipi_dsi_host_s *)dsi_host;
 	struct mipi_dsi_packet packet;
 	int cnt = 5000;
-	if(msg->type == MIPI_DSI_END_OF_TRANSMISSION){
+	/*When underflow happens, mipi's irq will be registered to rtl8730e_mipidsi_underflowreset callback to handle video mode frame done interrupt.
+	  only one type of interrupt can be enabled at any one time, either cmd mode interrupt or video mode interrupt, so we need to re-register mipi's
+	  irq cmd mode callback here
+	*/
+	if (g_rtl8730e_config_dev_s.underflow) {
```

Comment:
even if check g_rtl8730e_config_dev_s.underflow here, if the lcdc underflow come after this code, The mipi handler is already changed to rtl8730e_mipidsi_underflowreset, and mipi cmd can't get a tx done.